marklogic-data-hub-central/ui/src/components/explore/entity-type-display-settings-modal/entity-type-display-settings-modal.tsx

```diff
@@ -12,22 +12,20 @@ import {UserContext} from "@util/user-context";
 import {HubCentralConfigContext} from "@util/hubCentralConfig-context";
 import Highlighter from "react-highlight-words";
 import * as _ from "lodash";
-// ToDo: For use in DHFPROD-8869: Modify Explore Settings to Change hover properties (Structured properties only)
-// import EntityPropertyTreeSelect from "../../entity-property-tree-select/entity-property-tree-select";
-// import axios from "axios";
-// import {definitionsParser} from "@util/data-conversion";
-// import {Definition} from "../../../types/modeling-types";
+import EntityPropertyTreeSelect from "../../entity-property-tree-select/entity-property-tree-select";
+import axios from "axios";
+import {definitionsParser} from "@util/data-conversion";
+import {Definition} from "../../../types/modeling-types";
 
 type Props = {
   isVisible: boolean;
   toggleModal: (reloadData: boolean) => void;
   entityDefinitionsArray: any;
 };
-// ToDo: For use in DHFPROD-8869: Modify Explore Settings to Change hover properties (Structured properties only)
-// const DEFAULT_ENTITY_DEFINITION: Definition = {
-//   name: "",
-//   properties: []
-// };
+const DEFAULT_ENTITY_DEFINITION: Definition = {
```

Comment:
I see that we are defining DEFAULT_ENTITY_DEFINITION here as well as in the Side panel component. Let's define such default value variables in a config file (probably explore.config.ts under /config) and then just refer wherever needed.